ErsatzTV/Controllers/IptvController.cs

```diff
@@ -30,18 +33,26 @@ public class IptvController : ControllerBase
         _mediator = mediator;
         _logger = logger;
         _ffmpegSegmenterService = ffmpegSegmenterService;
+        try
```

Comment:
It is possible my compiler was the reason I had to implement a try catch statement instead of an if statement for the string being null.

Feel free to correct this to not implement try catch.

Reply:
I don't think request is typically used inside the controller, which may be the issue. I'm doing some minor reformatting and will move it out.